recipe/build_base.sh

```diff
@@ -520,7 +520,7 @@ if [[ ${PY_INTERP_DEBUG} == yes ]]; then
   rm ${PREFIX}/bin/python${VER}
   ln -s ${PREFIX}/bin/python${VERABI} ${PREFIX}/bin/python${VER}
   ln -s ${PREFIX}/lib/libpython${VERABI}.so ${PREFIX}/lib/libpython${VER}.so
-  ln -s ${PREFIX}/lib/libpython${VERABI}.so.1.0 ${PREFIX}/lib/libpython${VER}.so.1.0
+  ln -s ${PREFIX}/include/python${VERABI} ${PREFIX}/include/python${VER}
```

Comment:
@isuruf Thanks! How were you able to figure this out? I was having a really hard time

Reply:
Error message said
`Could NOT find PythonLibs (missing: PYTHON_INCLUDE_DIRS) (Required is at least version "3.11.0")`

Reply:
I see, I guess these kinds of errors become easy to address with experience, something that I lack right now.